recurring_ical_events.py

```diff
@@ -69,6 +69,7 @@ def time_span_contains_event(span_start, span_stop, event_start, event_stop,
         You can set it to True if you are sure you have timezones and
         date/datetime correctly or used make_comparable() before.
     """
+    print(span_start, span_stop, event_start, event_stop)
```

Comment:
isnt this a debug print?

Reply:
Thanks, yes. There is a test that prevents deploys with print statements in it. Well spotted. Thanks! It is removed now.